examples/pessimistic.rs

```diff
@@ -20,7 +23,7 @@ async fn main() {
     };
 
     // init
-    let client = Client::new_with_config(args.pd, config, None)
+    let client = Client::new_with_config(args.pd, config, TxnApiV1, None)
```

Comment:
`Client` itself can tell we are using TxnKV already, can we hide things like TxnApiV1, TxnApiV2, RawApiV1, RawApiV2. Just let user chose the appropriate client and tell he wants to use ApiV1 or ApiV2.

Reply:
We may reexport them in their own module, like: `raw::ApiV1` , `txn::ApiV1`